docs/API/fliplet-core.md

```diff
@@ -1010,7 +1010,17 @@ Fliplet.App.Analytics.Session.reset();
 
 Here's how you can use our powerful JS APIs to do some heavylifting for you and return aggregated records instead of having to group them manually when displaying charts for app analytics.
 
-Properties of the `get` function:
+**Note**: fetching aggregating logs is available both under the current app's namespace (`Fliplet.App`) and the apps namespace (`Fliplet.Apps`), please check for the difference between the two:
```

Comment:
I've restructured the sentence a little to:

1. Focus on the difference between the two
1. Prime the reader on why they should care about the differences

```
**Note**: fetching aggregating logs is available both under the namespace **for the current app** (`Fliplet.App`) and **for all apps** (`Fliplet.Apps`), each have different behaviors and parameter requirements.
```

Reply:
@tonytlwu thanks! I agree, just made the change.